examples/platform/linux/AppMain.cpp

```diff
@@ -269,6 +269,12 @@ LinuxCommissionableDataProvider gCommissionableDataProvider;
 
 chip::DeviceLayer::DeviceInfoProviderImpl gExampleDeviceInfoProvider;
 
+
+void ConnectivityChnagedtimerCallback(System::Layer *, void * callbackContext)
```

Comment:
"ChangedTimer"?

Reply:
I would like to change to void 
UpdateDnssdServiceCallback(System::Layer *, void * callbackContext)
{
    app::DnssdServer::Instance().StartServer();
}